proto/ibc/channel/channel.proto

```diff
@@ -29,40 +29,49 @@ message MsgChannelOpenTry {
 // MsgChannelOpenAck defines a msg sent by a Relayer to Chain A to acknowledge
 // the change of channel state to TRYOPEN on Chain B.
 message MsgChannelOpenAck {
-  string                                     port_id              = 1 [(gogoproto.customname) = "PortID", (gogoproto.moretags) = "yaml:\"port_id\""];
-  string                                     channel_id           = 2 [(gogoproto.customname) = "ChannelID", (gogoproto.moretags) = "yaml:\"channel_id\""];
-  string                                     counterparty_version = 3 [(gogoproto.moretags) = "yaml:\"counterparty_version\""];
-  bytes proof_try            = 4 [(gogoproto.moretags) = "yaml:\"proof_try\""];
-  uint64 proof_height = 5 [(gogoproto.moretags) = "yaml:\"proof_height\""];
-  bytes  signer       = 6 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
+  string                                     port_id               = 1 [(gogoproto.customname) = "PortID", (gogoproto.moretags) = "yaml:\"port_id\""];
+  string                                     channel_id            = 2 [(gogoproto.customname) = "ChannelID", (gogoproto.moretags) = "yaml:\"channel_id\""];
+  string                                     counterparty_version  = 3 [(gogoproto.moretags) = "yaml:\"counterparty_version\""];
+  string                                     connection_id         = 4 [(gogoproto.customname) = "ConnectionID", (gogoproto.moretags) = "yaml:\"connection_id\""];
```

Comment:
we shouldn't add these fields to the messages. Every channel is associated with a connection so we can just query it